diff.go

```diff
@@ -53,12 +53,25 @@ const (
 	// DiffInfoComponentExposeUpdated
 	DiffInfoComponentExposeUpdated DiffType = "component-expose-updated"
 
-	// DiffInfoComponentScaleUpdated
-	DiffInfoComponentScaleUpdated DiffType = "component-scale-updated"
+	// scale
+
+	// DiffInfoComponentScalePlacementUpdated
+	DiffInfoComponentScalePlacementUpdated DiffType = "component-scale-placement-updated"
+
+	// DiffInfoComponentScaleUp
+	DiffInfoComponentScaleUp DiffType = "component-scale-up"
```

Comment:
This is a great issue. Will think about it. This actually requires knowledge of the current environment. :confused: